src/main/java/com/example/msuser/exception/ErrorHandlerException.java

```diff
@@ -13,26 +12,19 @@
 
 @Slf4j
 @RestControllerAdvice
-public class ErrorHandler {
+public class ErrorHandlerException {
 
     @ExceptionHandler(Exception.class)
     @ResponseStatus(INTERNAL_SERVER_ERROR)
     public ExceptionResponse handle(Exception ex) {
         log.error("Exception: ", ex);
-        return new ExceptionResponse();
+        return new ExceptionResponse("Internal Server Error");
```

Comment:
move this to constant and dont use Internal Server Error. You can use for example, Unexpected error occurred